babel.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
Is this used for jest, as opposed to ts-jest? Have you found this configuration to work better? Just curious

Reply:
Yep, I remember just following the latest Jest getting started guide and that's what it recommended. Works great.